### PR TITLE
first pass regarding Admin page rework

### PR DESCRIPTION
**Pull request by emuus**
-add new screenshots
-merge sections about OS from config section to manage resources to avoid duplication
-split adding an OS and repo into two sections (UI and CLI)
-may need new screenshots

### CODE REVIEW

File: admin/en/source/pages/manage-resources/create-custom-os.rst [hunk 1]
```diff
@@ -2,35 +2,33 @@
...
 UForge provides a set of OS profiles. If you want to create another OS profile, you can create one using UForge's graphical user interface. 
 
 .. warning:: You can delete packages but we do not guarantee that your OS profile will be functional.
 
 To create a new OS profile:
 
-	#. Under the ``Administration`` tab, click ``OS Profiles``.
-	#. Click on the operating system you want to administer.  This will provide the current list of profiles this operating system has.
+	#. Under the ``Administration`` tab go to the ``Distributions`` tab.
+	#. Click on the operating system.  This will provide the current list of profiles this operating system has.
```
Comment (iamurri) [inline admin/en/source/pages/manage-resources/create-custom-os.rst]: When you click on an operating system in the Distributions tab, you land on the Repositories tab for that distribution rather than on the OS Profiles one, so I guess there's a step missing here

File: admin/en/source/pages/manage-resources/create-custom-os.rst [hunk 2]
```diff
@@ -2,35 +2,33 @@
...
-	#. Click on ``add os profile``.  
+	#. Click on ``add os profile`` in the top right.  
 
 		.. image:: /images/create-os-profile-button.png
 
 	#. Select the profile which will serve as the base for your custom OS profile.
 	#. Enter your profile name and click ``create``.
-	#. Enter a description (mandatory) and click ``save``.
-	#. Opionally, from the ``OS Packages`` tab you can select the package(s) you want to add to your OS profile and click ``save``. You can search for the package you want to add. The packages that you can add are listed in the top list. The bottom list is the list of packages that are currently part of the profile.
+	#. Select the package(s) you want to add to your OS profile. You can search for the package you want to add. The packages that you can add are listed in the top list. The bottom list is the list of packages that are currently part of the profile.
```
Comment (iamurri) [inline admin/en/source/pages/manage-resources/create-custom-os.rst]: The packages available are not listed unless you enter some search term in the search bar, maybe it would be good to explain

File: admin/en/source/pages/manage-resources/create-custom-os.rst [hunk 3]
```diff
@@ -40,14 +38,12 @@ If you have created a custom OS profile, you can edit it at any time.
...
-	2. Click on the operating system you want to administer.  This will provide the current list of profiles this operating system has.  
-	3. Click on the profile you want to edit.
+	1. Under the ``Administration`` tab, go to ``Distributions``.
+	2. Click on the operating system you want to administer.  Under the ``OS Profiles`` tab you will see a list of the current profiles this operating system has.  
+	3. Click on the OS profile you want to edit.
 
 		.. image:: /images/edit-os-profile.png
 
 		On this view, for Windows, you can see the distribution your OS profile is based on, its size, the date you created the OS profile in UForge (next to the OS profile name) as well as the last time it was edited.	
+	4. Select the package(s) you want to add to your OS profile. You can search for the package you want to add. The packages that you can add are listed in the top list. The bottom list is the list of packages that are currently part of the profile.
```
Comment (iamurri) [inline admin/en/source/pages/manage-resources/create-custom-os.rst]: I haven't seen a part where you explain how to edit an OS Profile's name and description, so I would have this section be named "To edit your OS profile's packages:" and maybe another one for name and description, or just a mention in this one

File: admin/en/source/pages/manage-resources/manage-os.rst [hunk 4]
```diff
@@ -63,18 +65,46 @@ Login to one of the UForge instances:
...
+
+	#. From the ``Administration`` page, go to the ``Distributions`` tab.
+	#. Click on ``add`` in the top left.
+	#. Select the distribution, version and architecture from the list.
+	#. Click ``add``.
+	#. You must add repositories to this distribution from the list of ``Available repositories`` by clicking on ``add``.
+
+		.. image:: /images/add-repo.png
+
+	#. You must populate the repositories. To do so, click on ``populate`` at the top right and follow the instructions in the pop-up window.
```
Comment (iamurri) [inline admin/en/source/pages/manage-resources/manage-os.rst]: maybe a sentence explaining where exactly this populate button is would be good (you must be in the edition view of the distribution, so after having clicked on it)

File: admin/en/source/pages/manage-resources/manage-os.rst [hunk 5]
```diff
@@ -63,18 +65,46 @@ Login to one of the UForge instances:
...
+
+		.. image:: /images/populate-popup.png
+
+	#. To add an OS profile (opional), go to the ``OS Profiles`` tab. You can create one by clicking on ``add os profile`` in the top right. Refer to :ref:`create-custom-os`.
```
Comment (iamurri) [inline admin/en/source/pages/manage-resources/manage-os.rst]: maybe specify that the population process provides base OS Profiles

File: admin/en/source/pages/manage-resources/manage-os.rst [hunk 6]
```diff
@@ -112,10 +142,27 @@ For example, to add CentOS 6.5 i386:
...
+		.. image:: /images/create-repo.png
+
+	.. note:: Check ``core repositories`` for all the default repositories of officially supported OSes (for a list of supported OSes, refer to :ref: uforge-supported-os-formats). Do not check this box for repositories that are not part of the core distribution, such as epel or VMware tools. When generating a machine image, packages tagged as ``core`` are installed first, before other packages.
+
+
 .. _updating-repo:
 
-Updating a Repository
----------------------
+Updating a Repository Using CLI
```
Comment (iamurri) [inline admin/en/source/pages/manage-resources/manage-os.rst]: You will also be able to update/edit a repository with the UI, Nicolas Tith is working on this feature if you need more information he can help you I think

Comment (emuus) [reply admin/en/source/pages/manage-resources/manage-os.rst]: @ntith will it be part of this sprint? (ie merged Thursday?)

Comment (iamurri) [reply admin/en/source/pages/manage-resources/manage-os.rst]: He told me it will

File: admin/en/source/pages/manage-resources/create-custom-os.rst (hunk 1 above)
Comment (emuus) [reply admin/en/source/pages/manage-resources/create-custom-os.rst]: ok

File: admin/en/source/pages/manage-resources/create-custom-os.rst (hunk 3 above)
Comment (emuus) [reply admin/en/source/pages/manage-resources/create-custom-os.rst]: ok

File: admin/en/source/pages/manage-resources/manage-os.rst (hunk 4 above)
Comment (emuus) [reply admin/en/source/pages/manage-resources/manage-os.rst]: ok I think if you follow the steps it should be ok, but since I can't see the button yet .... we can modify if needed

File: admin/en/source/pages/manage-resources/manage-os.rst (hunk 5 above)
Comment (emuus) [reply admin/en/source/pages/manage-resources/manage-os.rst]: ok

File: admin/en/source/pages/manage-resources/manage-os.rst (hunk 6 above)
Comment (emuus) [reply admin/en/source/pages/manage-resources/manage-os.rst]: ok done

File: admin/en/source/pages/manage-resources/manage-os.rst [hunk 7]
```diff
@@ -161,6 +163,17 @@ You can manage existing repositories and create new ones from the UForge UI, fro
...
 .. _updating-repo:
 
+Updating a Repository Using UI
+-------------------------------
+
+You can modify or update a repository from the UI, from the ``Administration`` page.
+
+	#. Go to the ``Repositories`` tab. The existing repositories will be listed.
+	#. Click on the pencil icon on the right of the repository you want to update.
+	#. In the pop-up window, modify the information and click ``update``.
```
Comment (iamurri) [inline admin/en/source/pages/manage-resources/manage-os.rst]: The edition of a repository isn't done in a popup window, but rather in a view similar to the create repository one

File: admin/en/source/pages/manage-resources/create-custom-os.rst [hunk 8]
```diff
@@ -16,13 +16,14 @@ To create a new OS profile:
...
 
+	#. Go to the ``OS Profiles`` tab.
 	#. Click on ``add os profile`` in the top right.  
 
 		.. image:: /images/create-os-profile-button.png
 
 	#. Select the profile which will serve as the base for your custom OS profile.
 	#. Enter your profile name and click ``create``.
-	#. Select the package(s) you want to add to your OS profile. You can search for the package you want to add. The packages that you can add are listed in the top list. The bottom list is the list of packages that are currently part of the profile.
+	#. Search for the packages you want to add. Select the package(s) you want to add to your OS profile. The packages that you can add are listed in the top list. The bottom list is the list of packages that are currently part of the profile.
```
Comment (iamurri) [inline admin/en/source/pages/manage-resources/create-custom-os.rst]: Maybe swap the two sentences like so:
Search for the packages you want to add. The packages that you can add are listed in the top list. Select the package(s) you want to add to your OS profile. The bottom list is the list of packages that are currently part of the profile.

File: admin/en/source/pages/manage-resources/create-custom-os.rst [hunk 9]
```diff
@@ -38,12 +39,17 @@ If you have created a custom OS profile, you can edit it at any time.
...
+
+		.. image:: /images/edit-os-profile-name.png
+
+	#. To modify the packages, click on the OS profile in the list.
 
 		.. image:: /images/edit-os-profile.png
 
 		On this view, for Windows, you can see the distribution your OS profile is based on, its size, the date you created the OS profile in UForge (next to the OS profile name) as well as the last time it was edited.	
-	4. Select the package(s) you want to add to your OS profile. You can search for the package you want to add. The packages that you can add are listed in the top list. The bottom list is the list of packages that are currently part of the profile. 
+	#. Search for the packages you want to add. Select the package(s) you want to add to your OS profile. The packages that you can add are listed in the top list. The bottom list is the list of packages that are currently part of the profile.
```
Comment (iamurri) [inline admin/en/source/pages/manage-resources/create-custom-os.rst]: Same as above:
Search for the packages you want to add. The packages that you can add are listed in the top list. Select the package(s) you want to add to your OS profile. The bottom list is the list of packages that are currently part of the profile.

File: admin/en/source/pages/manage-resources/manage-os.rst (hunk 7 above)
Comment (emuus) [reply admin/en/source/pages/manage-resources/manage-os.rst]: sorry, it gave this impression as there was no more the tabs "repo" distrib etc at the top. my bad.

File: admin/en/source/pages/manage-resources/create-custom-os.rst (hunk 8 above)
Comment (emuus) [reply admin/en/source/pages/manage-resources/create-custom-os.rst]: Disagree. The original order shows the actions 1. Search. then explains what you see. next action (maybe this could even be a seperate step) 2. Select the packages.

Comment (iamurri) [reply admin/en/source/pages/manage-resources/create-custom-os.rst]: ok

File: admin/en/source/pages/manage-resources/manage-os.rst (hunk 7 above)
Comment (iamurri) [reply admin/en/source/pages/manage-resources/manage-os.rst]: Ok, there actually was a bug there, thanks, we're working on fixing it :)

File: admin/en/source/pages/manage-resources/manage-os.rst [hunk 10]
```diff
@@ -63,18 +65,46 @@ Login to one of the UForge instances:
...
+Adding an OS Using the UI
+~~~~~~~~~~~~~~~~~~~~~~~~~
+
+In order to add an OS to an organisation you manage:
+
+	#. From the ``Administration`` page, go to the ``Distributions`` tab.
+	#. Click on ``add`` in the top left.
+	#. Select the distribution, version and architecture from the list.
+	#. Click ``add``.
+	#. You must add repositories to this distribution from the list of ``Available repositories`` by clicking on ``add``.
```
Comment (jbremond) [inline admin/en/source/pages/manage-resources/manage-os.rst]: We may explain that you should also create repository from this view if the repo is not listed in the available one

File: admin/en/source/pages/manage-resources/manage-os.rst [hunk 11]
```diff
@@ -112,10 +142,27 @@ For example, to add CentOS 6.5 i386:
 
 When adding CentOS, Debian and Red Hat, the major versions are automatically marked as Milestones when the distribution is added to the platform. For more information on Milestones, refer to :ref:`manage-milestone`.
 
+
+.. _creating-repo:
+
+Creating a Repository Using UI
```
Comment (jbremond) [inline admin/en/source/pages/manage-resources/manage-os.rst]: I am sorry for the bad news, but according to our last finding, we will deliver the whole feature without this feature (update a repository). In fact this is more complex than expected.
Would it be possible to remove this part for now, but keep it somewhere and reuse it when we will be ready ?

**Comment by ludop-usharesoft** [comment]
I took these commits and created a branch "adminRepo" with a extra one to rename "officiallySupported" by "coreRepository".
The new branch is here: https://github.com/usharesoft/appcenter-docs/tree/adminRepo
And the PR from the branch to master is here: https://github.com/usharesoft/appcenter-docs/pull/883

File: admin/en/source/pages/manage-resources/manage-formats.rst [hunk 12]
```diff
@@ -66,6 +66,8 @@ You can create your own image formats from the UForge GUI. To do so:
 	1. Create a target platform.
 	2. Create a target format.
 
+.. _target-platform-create:
```
Comment (jbremond) [inline admin/en/source/pages/manage-resources/manage-formats.rst]: I think there is no link between the feature of admin distribution/repository and this change ? If yes, should be done in a separate commit/PR

Comment (jbremond) [reply admin/en/source/pages/manage-resources/manage-formats.rst]: this comment apply for all the line in this file

File: admin/en/source/pages/manage-resources/manage-os.rst [hunk 13]
```diff
@@ -63,18 +65,48 @@ Login to one of the UForge instances:
...
+Adding an OS Using the UI
+~~~~~~~~~~~~~~~~~~~~~~~~~
+
+In order to add an OS to an organisation you manage:
+
+	#. From the ``Administration`` page, go to the ``Distributions`` tab.
+	#. Click on ``add`` in the top left.
+	#. Select the distribution, version and architecture from the list.
+	#. Click ``add``.
+	#. You must add repositories to this distribution from the list of ``Available repositories`` on the ``Repositories`` tab. The repositories that you can add are listed in the bottom list. The top list is the list of repositories that have been added.
```
Comment (jbremond) [inline admin/en/source/pages/manage-resources/manage-os.rst]: We don't speak here about the fact that we can now create a repository (and attach it automatically) from the distribution view. I think this is good to illustrate this use case here.

File: admin/en/source/pages/manage-resources/manage-formats.rst (hunk 12 above)
Comment (emuus) [reply admin/en/source/pages/manage-resources/manage-formats.rst]: @jbremond  as discussed over skype this file is impacted by a new admin banner. When adding the new screens, I took the opportunity to move the image to a better location and add a section title for better readability

File: admin/en/source/pages/manage-resources/manage-os.rst (hunk 13 above)
Comment (emuus) [reply admin/en/source/pages/manage-resources/manage-os.rst]: done !

**Comment by ludop-usharesoft** [comment]
All commits up to this point are integrated in https://github.com/usharesoft/appcenter-docs/pull/883/commits

**Comment by jbremond** [comment]
The new change is ok for me. Thank you @emuus

**Comment by Numerinico** [comment]
Done in #883
